srs_extension/extension.js

```diff
@@ -105,6 +105,80 @@ function activate(context) {
         }
     };
 
+    var insertReqTags = function () {
+        var text = loadText();
+
+        if (text) {
+            var nextReqTag = reqParser.getNextReq(text, reqPrefix, devId);
+
+            var editor = vscode.window.activeTextEditor;
+            var document = editor.document;
+
+            var endSnippet = ' **]**';
+
```

Comment:
Do you want to add any tests for these? Seems like maybe the part after loadtext can actually be tested easily.

Reply:
I figured out how to get int tests working so I added those and I have a few tests for this and the single requirement command :)

---
In reply to: [405872908](https://github.com/Azure/azure-c-build-tools/pull/18#discussion_r405872908) [](ancestors = 405872908)